doc/md_src_files/16_acknowledgements.md

```diff
@@ -1,17 +1,21 @@
 # Acknowledgements
 
-The authors thank the [contributors](https://github.com/CERTCC/SSVC/graphs/contributors) to the [SSVC project](https://github.com/CERTCC/SSVC) on Github as well as
-the following individuals for helpful comments on prior drafts (listed in alphabetical order): 
+The authors would first like to acknowledge the valuable contributions of previous authors who have worked on earlier versions
+of this report: Art Manion, Madison Oliver, and Deana Shick.
+
+The authors thank the [contributors](https://github.com/CERTCC/SSVC/graphs/contributors) to the
+[SSVC project](https://github.com/CERTCC/SSVC) on Github as well as the following individuals for helpful comments on
+prior drafts (listed in alphabetical order): 
 Muhammad Akbar, 
 Will Dormann,
 Manish Gaur,
 Ralph Langer,
 David Oxley
 Dale Peterson,
-Deana Shick,
```

Comment:
I removed Deana from the reviewer list since she's in the newly-added previous authors list.